packages/sitecore-jss-nextjs/src/middleware/redirects-middleware.ts

```diff
@@ -135,7 +135,7 @@ export class RedirectsMiddleware {
       }
     };
 
-    const response = await createResponse();
+    const response = await createResponse(res);
```

Comment:
You don't need to provide `res` here, cause it's available in `handler arguments`